signalfx-tracing/signalfx-nodejs-tracing/README.md

```diff
@@ -13,6 +13,7 @@ instrumentation configuration options, please see the
 ## Libraries and Framework Examples
 
 - [Express](./express)
+- [Hapi + Postgres](./hapi_pg)
```

Comment:
I would list this as an Hapi example, and have another line about Postgres that points people back at the Hapi example, to be clear about the fact that those don't necessarily have to be used together when instrumenting.